pkg/lua/Makefile

```diff
@@ -0,0 +1,13 @@
+PKG_NAME=lua
+PKG_URL=https://github.com/lua/lua.git
+PKG_VERSION=v5-3-4
```

Comment:
This should pointing to a commit. See e.g [here](https://github.com/RIOT-OS/RIOT/blob/master/pkg/jerryscript/Makefile#L3)